.github/workflows/CICD.yml

```
@@ -66,12 +69,14 @@ jobs:
               - 'phpunit.xml'
               - 'phpunit.ci.xml'
               - 'phpunit.pgsql.xml'
+            lang:
+              - 'lang/**'
 
   php_syntax_errors:
     name: 0️⃣ PHP 8.4 - Syntax errors
     runs-on: ubuntu-latest
     needs: changes
-    if: github.event_name != 'pull_request' || needs.changes.outputs.php == 'true'
+    if: github.event_name != 'pull_request' || needs.changes.outputs.php == 'true' || needs.changes.outputs.lang == 'true'
```

actionlint (1.7.12)

[error] property "lang" is not defined in object type {js: string; php: string}

(expression)